arviz/stats/diagnostics.py

```diff
@@ -1,29 +1,28 @@
 """Diagnostic functions for ArviZ."""
+import itertools
 import warnings
 
 import numpy as np
 import pandas as pd
 from scipy.signal import fftconvolve
+import xarray as xr
 
-
-from ..utils import trace_to_dataframe, get_varnames
+from ..utils.xarray_utils import convert_to_dataset
 
 
 __all__ = ['effective_n', 'gelman_rubin', 'geweke']
 
 
-def effective_n(trace, varnames=None, round_to=2):
+def effective_n(data, *, var_names=None):
```

Comment:
I didnt know you can use `*` as input - what does it do?

Reply:
It prevents people from using positional arguments